src/org/opendatakit/briefcase/transfer/TransferForms.java

```diff
@@ -36,34 +36,36 @@
 public class TransferForms implements Iterable<FormStatus> {
   private List<FormStatus> forms;
   private Map<String, FormStatus> formsIndex = new HashMap<>();
+  private final Map<String, String> lastPullCursorsByFormId;
```

Comment:
The comment for this commit, 
`Add somewhere to store in memory cursors into TransferForms`
would be more readable as 
`... store in-memory cursors ...`